x/ibc/07-tendermint/misbehaviour.go

```diff
@@ -63,23 +73,25 @@ func checkMisbehaviour(
 		)
 	}
 
-	if !bytes.Equal(consensusState.ValidatorSetHash, evidence.FromValidatorSet.Hash()) {
-		return errors.New(
-			"the consensus state's validator set hash doesn't match the evidence's one",
-		)
+	// NOTE: header height and commitment root assertions are checked with the
```

Comment:
Can/should we re run those checks here just to be safe?

Reply:
We could, but there's no need to as the validation is already checked. I'd always pass